wandb/sdk/artifacts/artifacts_cache.py

```diff
@@ -126,6 +164,13 @@ def cleanup(self, target_size: int, remove_temp: bool = False) -> int:
 
             total_size -= stat.st_size
             bytes_reclaimed += stat.st_size
+
+        if total_size >= target_size:
+            wandb.termerror(
```

Comment:
lgtm

**(suggestion)**: Do you think this warning should probably only be given if `temp_size > 0`? In case there are no temporary files, is there any other workaround that could be suggested here?

Reply:
As long as target_size >= 0, we should be able to reach it unless temp files prevent us from doing so, since we're empowered to delete all other files, so I don't think that condition can arise (temp_size == 0 and total_size > target_size)